src/goddard/gd_math.c

```diff
@@ -178,7 +178,7 @@ void gd_create_origin_lookat(Mat4f *mtx, struct GdVec3f *vec, f32 roll) {
 
     gd_set_identity_mat4(mtx);
     if (hMag != 0.0f) {
-        invertedHMag = 1.0f / hMag;
+        invertedHMag = swiDivide(1.0f, hMag);
```

Comment:
`swiDivide` doesn't support floats. Also, changes to the game code should be `ifdef`ed to preserve the N64 build.